lib/osf-components/addon/components/search-result-card/template.hbs

```diff
@@ -100,8 +98,15 @@
         <CpPanel local-class='cp-panel' id={{secondaryMetadataPanelId}} @open={{this.isOpenSecondaryMetadata}} as |panel|>
             <panel.body>
                 <hr>
-                {{component this.secondaryMetadataComponent result=@result}}
+                {{#if this.isUserResultCard}}
+                    <SearchResultCard::UserSecondaryMetadata
+                        @user={{this.osfUser}}
+                        @fetchTask={{this.getOsfUserModel}}
+                    />
+                {{else}}
+                    {{component this.secondaryMetadataComponent result=@result}}
+                {{/if}}
```

Comment:
could the user-fetching live on the `user-secondary-metadata` component instead? then it could have the same interface as each other `secondaryMetadataComponent`